commander/test/bootstrapping/commands/generator.spec.ts

```diff
@@ -108,19 +120,53 @@ describe('forging', () => {
 			});
 		});
 
+		describe('when use-status-value is used', () => {
+			it('should invoke action with given address and user provided password', async () => {
+				invokeMock.mockResolvedValue({
+					status: [
+						{ address: 'myAddress', height: 20, maxHeightGenerated: 3, maxHeightPrevoted: 4 },
+					],
+				});
+
+				await EnableCommand.run(
+					['myAddress', '--use-status-value', '--password=my-password'],
+					config,
+				);
+
+				expect(invokeMock).toHaveBeenCalledWith('generator_updateStatus', {
+					address: 'myAddress',
+					enable: true,
+					password: 'my-password',
+					height: 20,
+					maxHeightGenerated: 3,
+					maxHeightPrevoted: 4,
+				});
+				expect(stdout[0]).toMatch('Enabled block generation for myAddress');
+			});
+		});
+
 		describe('when action is successful', () => {
 			it('should invoke action with given address and user provided password', async () => {
-				await EnableCommand.run(['myAddress', '10', '10', '1', '--password=my-password'], config);
-				expect(stdout[0]).toMatch(actionResult);
+				await EnableCommand.run(
+					[
+						'myAddress',
+						'--height=10',
+						'--max-height-generated=10',
+						'--max-height-prevoted=1',
+						'--password=my-password',
+					],
+					config,
+				);
+				expect(stdout[0]).toMatch('Enabled block generation for myAddress');
 			});
 		});
 
 		describe('when action fail', () => {
 			it('should log the error returned', async () => {
 				when(invokeMock)
-					.calledWith('generator_setStatus', {
-						address: 'myFailedEnabledAddress',
-						enabled: true,
+					.calledWith('generator_updateStatus', {
+						address: 'myFailedenableAddress',
```

Comment:
Probably automated replace from `enabled` to `enable` messed up the camelCase here.

This is such an irrelevant test value, but still `myFailedEnableAddress` is more readable 😊